src/constants/test1/AELF.ts

```diff
@@ -1,12 +1,14 @@
 export const CHAIN_INFO = {
   chainId: 'AELF',
   exploreUrl: 'https://explorer-test.aelf.io/',
-  rpcUrl: 'http://192.168.66.61:8000',
+  rpcUrl: 'https://aelf-test-node.aelf.io',
```

Comment:
Is this correct? Seems to be testnet and not test1

Reply:
is this file changed?

Reply:
Please check now.